composer.json

```
@@ -34,12 +34,12 @@
         "ext-pcre": "*"
     },
     "require-dev": {
-        "phpunit/phpunit": "^8.5 || ^9.6",
+        "phpunit/phpunit": "^10.1",
```

```suggestion
        "phpunit/phpunit": "^10.1 || ^11",
```

should we add v11 as well?